CPAC/anat_preproc/utils.py

```diff
@@ -3,6 +3,30 @@
 
 from CPAC.pipeline import nipype_pipeline_engine as pe
 
+def pad(cropped_image_path, target_image_path):
+    from numpy import asanyarray, zeros_like
+    from nibabel import load, save, Nifti1Image
+    from os import path, getcwd
+
+    cropped_image = asanyarray(load(cropped_image_path).dataobj)
+    target_image = asanyarray(load(target_image_path).dataobj)
+
+    # Taking 1 slice to calculate the z dimension shift from top
+    row =target_image.shape[0]//2
+    column= target_image.shape[1]//2
+    a = cropped_image[row, column, :]
+    b = target_image[row, column, :]
+
+    for z_shift in range(len(b) - len(a) + 1):
+        if (b[z_shift:z_shift+len(a)] == a).all():
+            break
```

Comment:
What's this loop for?

Reply:
Ah, got it. I think making a separate function might be more Pythonic, but it's totally fine here if we don't need to repeat it elsewhere